### ISSUE TITLE
Add multiline support to @slot description

### ISSUE THREAD

**Issue by abdul-alhasany** [vue-styleguidist/vue-styleguidist#1441]
**The problem**
Currently when adding a description to a `@slot` tag, the script will only parse the first line and ignores the rest of the description.

**Proposed solution**
Parse multiline description. It would be great if that is applicable to all tags.

**Alternative solutions**
I considered using `addTemplateHandlers` but did not go through with it for two reasons:
- The task seemed too complicated to be handled by `addTemplateHandlers`. I don't know much about the base code to implement such feature.
- I think such feature should be part of core. Since this script is used for documentation, it would be a common occurrence to write multiline description.

**Additional context**
Beside being easy to read description, the linter was complaining the line is too long and it needs to broken down into smaller sizes.

Thanks.

**Comment by yellowsunday** [comment]
Multiple lines used to be supported in `@slot` descriptions: my styleguide has several long slot descriptions including some with code blocks, so I noticed when trying to upgrade to the latest version.

The old behaviour can be seen in vue-styleguidist 4.44.2 with vue-docgen-api 4.44.2.
It stopped working with vue-docgen-api 4.46.0 https://github.com/vue-styleguidist/vue-styleguidist/releases/tag/vue-docgen-api%404.46.0  which is actually mentioned in that update. Perhaps the previous multiline support was an accidental feature?

Is there any workaround possible to get these long slot descriptions displaying again?

**Comment by duclet** [comment]
Yeah, my slots cross multiple lines too. I don't know why the expectation is for slots to only be 1 line and there is no mention of this being a bug before. Regardless, this is a backward incompatible change and really should have been a major version bump at least. I pin versions in most of my projects so probably wouldn't affect me for now but......
